entity/person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..10_000)
+    @name = name
+    @age = age
+    @permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    age > 18
+  end
```

Comment:
- [ ] Kindly use the instance variable `@age`, instead of just `age` to avoid errors.

- [ ] This method should check if the age is greater than OR EQUAL to 18. Kindly use the appropriate sign: `>=`